learning_journal/setup.py

```diff
@@ -3,18 +3,22 @@
 from setuptools import setup, find_packages
 
 here = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(here, 'README.txt')) as f:
+with open(os.path.join(here, 'README.md')) as f:
     README = f.read()
 with open(os.path.join(here, 'CHANGES.txt')) as f:
     CHANGES = f.read()
 
 requires = [
     'pyramid',
     'pyramid_jinja2',
-    'ipython',
-    'pyramid_ipython',
     'pyramid_debugtoolbar',
+    'pyramid_tm',
+    'SQLAlchemy',
+    'transaction',
+    'zope.sqlalchemy',
     'waitress',
+    'ipython',
+    'pyramid_ipython'
```

Comment:
You probably don't need to install iPython and pyramid_ipython on heroku.  Maybe these should be in a "development" extra?  Like the way the testing stuff is in a "testing" extra?